src/app/containers/CpsAssetPageMain/index.test.jsx

```diff
@@ -66,25 +87,24 @@ describe('CpsAssetPageMain', () => {
       cpsAssetPreprocessorRules,
     );
 
+    await matchSnapshotAsync(createMediaAssetPage({ pageData }));
+  });
+
+  it('should match snapshot for MAP - hidden timestamp', async () => {
+    const pageData = await preprocessor(
+      pidginPageData,
+      cpsAssetPreprocessorRules,
+    );
+
+    const pageDataWithHiddenTimestamp = dissocPath(
+      ['metadata', 'options', 'allowDateStamp'],
+      pageData,
+    );
```

Comment:
Had a quick look at https://ramdajs.com/docs/#dissocPath and I'm wondering if this could be simpler, with something like `pageData.options.allowDateStamp = false`, maybe it would make sense in a expectance test rather than the snapshot, would be interested to see what other reviewers think

Reply:
`pageData.options.allowDateStamp` doesn't have the null protection that dissocPath does.

I've wondered this for a while - I can't actually find a succinct way to safely set a deeply nested value with ramda.  There is ramda.set, but that seems to require a lens, which I never need, and it makes the code very verbose - any suggestion @jroebu14 ?

Reply:
I think `dissocPath` works well in this case. It's quite simple compared to null-checking the path, and it's for a test so it's performance is acceptable.

Reply:
@ryanmccombe it looks like the safest way to set the value when it's deeply nested is [assocPath](https://ramdajs.com/docs/#assocPath) rather than `dissocPath`. Seems to me that `dissocPath` is unsetting the value rather than swapping it from `true` to` false`. IMO We should use `assocPath` to set the value to false. 

EG: `assocPath(['metadata', 'options', 'allowDateStamp'], false, pageData)`

Reply:
+:1 Updated